exercises/wordy/src/example.c

```diff
@@ -0,0 +1,75 @@
+#include "wordy.h"
+#include <string.h>
+#include <ctype.h>
+
+typedef struct {
+   const char *str;
+   const size_t len;
+
+} map_t;
+
+static const map_t q_init = { "What is", 7 };
+
+#define COMMANDS \
+        CMD(+, "plus", 4) \
+        CMD(-, "minus", 5) \
+        CMD(*, "multiplied by", 13) \
+        CMD(/, "divided by", 10)
+
+static void skip_space(const char **str)
+{
+   while (*(*str) == ' ')
+      (*str)++;
+}
+
+static bool get_int(const char **str, int *i)
+{
+   skip_space(str);
+   size_t int_len = 0;
+   *i = 0;
+   int sign = 1;
+   if (*(*str) == '-') {
+      sign = -1;
+      (*str)++;
+   }
+   while (*(*str) && isdigit(*(*str))) {
+      int_len++;
+      (*i) = (*i) * 10 + *(*str) - '0';
+      (*str)++;
+   }
+   (*i) *= sign;
+   return (int_len != 0);
+}
```

Comment:
Is there a reason to not use something like `sscanf` here?

Reply:
Well, no meaningful reason ... :)
1. It was fun to implement it myself.
2. Also, until now I did not know how to increment the string pointer after scanning, but your comment motivated me to search and learn about this.